.gitignore

```diff
@@ -1,2 +1,3 @@
 target
-.flatpak-builder
\ No newline at end of file
+.flatpak-builder
+aardvark-app/src/config.rs
```

Comment:
The `config.rs` file is ignored but still checked in into the file tree, is this intentional?

https://github.com/p2panda/aardvark/blob/jsparber/use_trait/aardvark-app/src/config.rs

Reply:
yeah, i was wondering why adding it to `.gitigore` wasn't enough. It needs to be deleted as well.